// gatsby/dist/utils/flags.js
"use strict";

exports.__esModule = true;
exports.default = void 0;
// Does this experiment run for only builds
const activeFlags = [{
  name: `FAST_DEV`,
  env: `GATSBY_EXPERIMENTAL_FAST_DEV`,
  command: `develop`,
  telemetryId: `FastDev`,
  experimental: false,
  description: `Enable all experiments aimed at improving develop server start time`,
  includedFlags: [`DEV_SSR`, `QUERY_ON_DEMAND`, `LAZY_IMAGES`]
}, {
  name: `DEV_SSR`,
  env: `GATSBY_EXPERIMENTAL_DEV_SSR`,
  command: `develop`,
  telemetryId: `DevSsr`,
  experimental: false,
  description: `SSR pages on full reloads during develop. Helps you detect SSR bugs and fix them without needing to do full builds.`,
  umbrellaIssue: `https://github.com/gatsbyjs/gatsby/discussions/28138`
}, {
  name: `QUERY_ON_DEMAND`,
  env: `GATSBY_EXPERIMENTAL_QUERY_ON_DEMAND`,
  command: `develop`,
  telemetryId: `QueryOnDemand`,
  experimental: false,
  description: `Only run queries when needed instead of running all queries upfront. Speeds starting the develop server.`,
  umbrellaIssue: `https://github.com/gatsbyjs/gatsby/discussions/27620`,
  noCi: true
}, {
  name: `LAZY_IMAGES`,
  env: `GATSBY_EXPERIMENTAL_LAZY_IMAGES`,
  command: `develop`,
  telemetryId: `LazyImages`,
  experimental: true,
  description: `Don't process images during development until they're requested from the browser. Speeds starting the develop server.`,
  umbrellaIssue: `https://github.com/gatsbyjs/gatsby/discussions/27603`
}];
var _default = activeFlags;
exports.default = _default;
